### PR TITLE
Adding dependent workflow to run after build

### PR DESCRIPTION
**Pull request by Alan-Jowett**
Testing adding a new dependent workflow.

### CODE REVIEW

**Review by saxena-anurag — APPROVED** [review]

File: .github/workflows/driver_test.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    workflows: ["MSBuild"]
+    types:
+      - completed
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        configurations: [Debug, Release]
+    runs-on: windows-latest
```
Comment (shankarseal) [inline .github/workflows/driver_test.yml:23]: is there any way to specify server?

**Review by shankarseal — APPROVED** [review]

File: .github/workflows/driver_test.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+# SPDX-License-Identifier: MIT
+
+# For documentation on the github environment, see
+# https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners
+#
+# For documentation on the syntax of this file, see
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Kernel_Test
+
+# Run this work flow when the "MSBuild workflow completes"
```
Comment (dthaler) [inline .github/workflows/driver_test.yml:11]: "workflow" (one word)

File: .github/workflows/driver_test.yml (hunk 1 above)
Comment (dthaler) [reply .github/workflows/driver_test.yml:23]: According to the link in line 5, "The windows-latest label currently uses the Windows Server 2019 runner image."

**Comment by dthaler** [comment]
What does this PR do?  I don't see anything in the Checks that looks like it installed ebpf.  Can you provide more info to help me review this PR?

**Comment by Alan-Jowett** [comment]
> What does this PR do? I don't see anything in the Checks that looks like it installed ebpf. Can you provide more info to help me review this PR?

This is the bare-bones version of the check that does the following:
1) Download the artifacts from the previous run.
2) Invoke the "install-ebpf.bat", which installs and starts the eBPF driver + service.

**Pull request merged**